### PR TITLE
Send html email feedback async

### PR DESCRIPTION
**Pull request by huyennguyen88**
- Send async email using JavaMail, task scheduling and anotation `@Async`
- Create content of html email using thymeleaf

### CODE REVIEW

**Comment by huyennguyen88** [comment]
ready

File: src/main/java/com/triplet/service/impl/EmailServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+	    // Prepare the context and variable
+		User user = feedback.getUser();
+	    Context ctx = new Context(locale);
+	    ctx.setVariable("id",user.getId());
+	    ctx.setVariable("username", user.getUsername());
+	    ctx.setVariable("fullname", user.getFullname());
+	    ctx.setVariable("address", user.getAddress());
+	    ctx.setVariable("phone", user.getPhone());
+	    ctx.setVariable("email", user.getEmail());	
+	    ctx.setVariable("content", feedback.getContent());
```
Comment (vanvtt-0952) [inline src/main/java/com/triplet/service/impl/EmailServiceImpl.java]: Tách riêng thèn này thành 1 method private đi bé

Comment (huyennguyen88) [reply src/main/java/com/triplet/service/impl/EmailServiceImpl.java]: Dạ em sửa rồi 👍

**Comment by huyennguyen88** [comment]
ready

**Pull request merged**